examples/apps/collaborative-textarea/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fluid-example/collaborative-textarea",
-  "version": "2.0.0-internal.1.3.0",
+  "version": "2.0.0-internal.1.5.0",
```

Comment:
Directly from 1.3 to 1.5?

Reply:
Yeah, we're doing an unusual release of 1.4 and skipping 1.3. Short version is that we have prerelease versions that get picked up by the semver ranges when they shouldn't. Bumping moves us past the "poisoned" version series.

Reply:
Added more details to the description.